person.rb

```diff
@@ -1,27 +1,29 @@
-require './corrector.rb'
+require './corrector'
 
 class Person
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
   attr_reader :id
 
-  def initialize(age, name = "Unknown", parent_permission = true)
+  def initialize(age, name = 'Unknown', parent_permission: true)
     @id = Random.rand(1..1000)
-    @corrector = Corrector.new()
+    @corrector = Corrector.new
     @age = age
     @name = name
     @parent_permission = parent_permission
+    @rentals = []
```

Comment:
- [ ] Kindly add a method for adding rentals in this class to achieve the has-many association between a person and rentals.